overlord/configstate/configcore/refresh.go

```diff
@@ -40,6 +40,19 @@ func init() {
 	supportedConfigurations["core.refresh.rate-limit"] = true
 }
 
+func reportOrResetInvalidManageRefreshes(tr config.Conf, optName string) error {
+	// check if the option is set as part of transaction changes; if not than
+	// it's already set in the config state.
+	for _, k := range tr.Changes() {
+		if k == "core."+optName {
+			return fmt.Errorf("cannot set schedule to managed")
+		}
+	}
+	tr.Set("core", optName, nil)
+	tr.State().Warnf(`core config option %s="managed" cannot be used with current snap setup and has been reset`, optName)
```

Comment:
I understand that the user is at the terminal at this point otoh is a bit strange to mention an option that is not getting touched here,   maybe this warning needs to be in autorefresh.go inside/near  refreshScheduleManaged perhaps?